bigquery_etl/cli/backfill.py

```diff
@@ -176,15 +177,20 @@ def validate(
     project_id,
 ):
     """Validate backfill.yaml files."""
-    backfills_dict = get_backfill_file_to_entries_map(
+    backfills_dict = get_qualifed_table_name_to_entries_map(
         sql_dir, project_id, qualified_table_name
     )
 
-    for backfill_file in backfills_dict:
+    for qualified_table_name in backfills_dict:
         try:
+            backfill_file = get_backfill_file_from_qualified_table_name(
+                sql_dir, qualified_table_name
+            )
             validate_file(backfill_file)
```

Comment:
I'd be nice to be able to validate all selected backfill confligs and see all that fail without having to re-run the app to just find out there is another broken config.